cmd/collector/main.go

```diff
@@ -36,7 +36,6 @@ import (
 	ss "github.com/jaegertracing/jaeger/plugin/sampling/strategystore"
 	"github.com/jaegertracing/jaeger/plugin/storage"
 	"github.com/jaegertracing/jaeger/ports"
-	tCollector "github.com/jaegertracing/jaeger/tchannel/collector/app"
```

Comment:
Is this issue also dealing with removing the `tchannel` endpoint from the collector? End user applications will not be able to send spans directly to the collector? This is a breaking change and should be documented

Reply:
End applications are not using this port. This is used only by the agent.